tests/istio_testcases/expected_output/connectivity_map_configs_w_sidecars_different_outbound_mode.txt

```diff
@@ -1,12 +1,12 @@
 For connections of type TCP, final fw rules for query: connectivity-allow-any-configs-different-outbound-modes, config: sidecar-with-selector-allows-any:
 src: 0.0.0.0/0 dst_ns: [default] dst_pods: [*] conn: All connections
-src_ns: [default] src_pods: [*] dst: *.newrelic.com conn: TCP 443
```

Comment:
Why are these differences?

Reply:
currently (in this PR) we don't support port field in service-entry, so we should see all conns to dnsEntry peers as `All connections`.

this file was on master branch before this PR by mistake (was not used/referenced).
In coming PRs we are planning to support the ports field , then we'll see the TCP conn again